editor/export_template_manager.cpp

```diff
@@ -499,7 +499,7 @@ void ExportTemplateManager::_notification(int p_what) {
 
 	if (p_what == NOTIFICATION_VISIBILITY_CHANGED) {
 		if (!is_visible_in_tree()) {
-			print_line("closed");
+			print_line(TTR("closed"));
```

Comment:
Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.